src/components/ghost/__tests__/Ghost.stories.test.tsx

```diff
@@ -0,0 +1,9 @@
+describe('components/ghost/Ghost', () => {
+    test.each([['components-ghost--regular'], ['components-ghost--without-animation']])(
```

Comment:
@sahiga Should the animated one be excluded from the screenshot test? It passed multiple times when I tested it, but wanted to double check.

Reply:
Good question. I typically exclude animated stories from visual regression tests, but since this one has passed multiple times, you can leave it.